apps/docs/src/app/platform/component-docs/platform-slider/examples/form-field/slider-form-field-example.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+    selector: 'fdp-slider-form-field-example',
+    templateUrl: './slider-form-field-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
+    styles: [
+        `
+            .slider-example .fd-container.fd-form-layout-grid-container .fd-row .fd-col {
+                overflow: initial !important;
```

Comment:
`initial` also will not be supported by IE ;) can we not just keep `auto`?

Reply:
It not helped
Will try `visible`